src/customScreens.ts

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
Can we rename this to screenMapping.ts or something?  The implication of 'custom' makes me uneasy. I don't want people to think this is the wild west and the way to change the color theme is to create a whole new 'custom' screen for it.

Reply:
Sure, no problem!